packages/2019-housing/src/components/BlackPopulationChange/BlackPopulationChangeVisualization.js

```diff
@@ -5,13 +5,18 @@ import { scaleQuantize, extent } from "d3";
 
 import {
```

Comment:
In `packages/2019-housing/src/components/AduDistributions/index.js`, the eslint-disable is not working for init, needs to be repositioned